### PR TITLE
Webhooks

### PR DESCRIPTION
**Pull request by soyombo-baterdene**

### CODE REVIEW

File: src/data/utils.ts [hunk 1]
```diff
@@ -1035,3 +1036,27 @@ export const s3Stream = async (key: string, errorCallback: (error: any) => void)
 
   return stream;
 };
+
+/**
+ * Send to webhook
+ */
+
+export const messageSendtoWebhook = async (message: any, type: string) => {
```
Comment (batamar) [inline src/data/utils.ts:1044]: fix camel case

File: src/data/utils.ts [hunk 2]
```diff
@@ -1035,3 +1035,30 @@ export const s3Stream = async (key: string, errorCallback: (error: any) => void)
...
+  }
+
+  for (const webhook of webhooks) {
+    if (!webhook.url || webhook.url.length === 0) {
+      continue;
+    }
+    sendRequest({
+      url: webhook?.url,
+      headers: {
+        'Content-Type': 'application/json',
```
Comment (munkhorgil) [inline src/data/utils.ts:1057]: remove "Content-Type" it's defined in sendRequest by default

File: src/db/models/Webhook.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        throw new Error('Webhook not found');
+      }
+
+      return webhook;
+    }
+
+    public static async getWebHooks() {
+      const webhooks = await Webhooks.find({});
+
+      return webhooks;
```
Comment (munkhorgil) [inline src/db/models/Webhook.ts:46]: return above

File: src/data/resolvers/mutations/integrations.ts [hunk 4]
```diff
@@ -179,9 +190,21 @@ const integrationMutations = {
...
   },
 
-  async integrationsEditCommonFields(_root, { _id, name, brandId, channelIds }, { user }) {
+  async integrationsEditCommonFields(_root, { _id, name, brandId, channelIds, data }, { user }) {
     const integration = await Integrations.getIntegration(_id);
-    const updated = Integrations.updateBasicInfo(_id, { name, brandId });
+
+    const doc: any = { name, brandId, data };
+
+    switch (integration.kind) {
```
Comment (munkhorgil) [inline src/data/resolvers/mutations/integrations.ts:198]: use if statement

**Pull request merged**